### DISCUSSION TITLE
Hook Types?

### DISCUSSION THREAD

**Discussion by aaronmgdr** [typicode/husky#1311] [Q&A] [answered]
Is there any where that documents what hooks are possible with husky? 

I know there is pre-commit and i think pre-push. 

are there more? 

I have search [the docs](https://typicode.github.io/husky/#features) and readme.

**Comment by typicode** [comment, marked as answer]
it's here https://typicode.github.io/husky/ click on [all client-side Git hooks](https://git-scm.com/docs/githooks) :)

**Reply by aaronmgdr** [reply]
of course i should have realized it was githooks :)
